html/pfappserver/root/static.alt/src/views/Nodes/_components/NodeView.vue

```diff
@@ -3,12 +3,15 @@
   <b-form @submit.prevent="save()">
     <b-card no-body>
       <b-card-header>
-        <b-button-close @click="close"><icon name="times"></icon></b-button-close>
+        <b-button-close @click="close" v-b-tooltip.hover.left :title="$t('Close [ESC]')"><icon name="times"></icon></b-button-close>
```

Comment:
I propose an [initial delay](https://bootstrap-vue.js.org/docs/directives/tooltip/#directive-syntax-and-usage) for the tooltip:

` v-b-tooltip.hover.left.d300 `